src/test/java/io/supertokens/storage/mysql/test/ConfigTest.java

```diff
@@ -258,6 +258,8 @@ public void testAddingTableNamePrefixWorks() throws Exception {
         MySQLConfig config = Config.getConfig((Start) StorageLayer.getStorage(process.getProcess()));
 
         assertEquals("change in KeyValueTable name not reflected", config.getKeyValueTable(), "key_value_table");
+        assertEquals("change in AccessTokenSigningKeysTable name not reflected", config.getAccessTokenSigningKeysTable(),
+                "some_prefix_session_access_token_signing_keys");
```

Comment:
Please remove this. It's not needed, and it's not testing what you intended it to test either.